src/app.rs

```diff
@@ -57,6 +57,8 @@ use crate::{
     tab::{self, HeadingOptions, ItemMetadata, Location, Tab, HOVER_DURATION},
 };
 
+static REFRESH_VIEW_DELAY_MS: u64 = 20; 
```

Comment:
The delay value was a totally arbitrary decision.

Reply:
I guess in a 60Hz display you need at least 16.6 ms , so you will see at least one frame  and more that one in higher refresh rate displays assuming the execution of the rescan of the view contents is instantaneous.